.github/workflows/ci.yml

```diff
@@ -75,7 +81,7 @@ jobs:
           # activate our env again
           source $CONDA/etc/profile.d/conda.sh
           conda activate ci
-          coverage run -m pytest -v
+          coverage run -m pytest -v 
```

Comment:
sorry for being picky, but again some unnecessary space that triggers a change of a line that was not really changed.

Reply:
done